tests/cluster/tests/00-base.tcl

```diff
@@ -59,6 +59,43 @@ test "Sanity for CLUSTER COUNTKEYSINSLOT" {
     assert {$reply eq 0}
 }
 
+test "Sanity for CLUSTER BUMPEPOCH" {
+    set reply [R 0 CLUSTER BUMPEPOCH]
+    assert_match {BUMPED*} $reply
+}
+
+test "Sanity for CLUSTER COUNT-FAILURE-REPORTS" {
+    set id [R 0 CLUSTER MYID]
+    set reply [R 0 CLUSTER COUNT-FAILURE-REPORTS $id]
+    assert {[string is integer $reply]}
+}
+
+test "Sanity for CLUSTER KEYSLOT" {
+    set reply [R 0 CLUSTER KEYSLOT foo]
+    assert {[string is integer $reply]}
+}
+
+test "Sanity for CLUSTER SAVECONFIG" {
+    set reply [R 0 CLUSTER SAVECONFIG]
+    assert {$reply eq "OK"}
+}
+
 test "It is possible to write and read from the cluster" {
     cluster_write_test 0
 }
+
+test "Sanity for CLUSTER FORGET" {
+    set id [R 0 CLUSTER MYID]
+    catch {[R 0 CLUSTER FORGET $id]} e
+    assert_match {*can't forget myself*} $e
+
+    set id [R 1 CLUSTER MYID]
+    set reply [R 0 CLUSTER FORGET $id]
```

Comment:
For 60 seconds Node 1 won't acknowledge node 0 exists because of the blacklist timeout. Might want to call `CLUSTER RESET` on node 0 and wait for convergence. Given that nothing happens after this is should be okay, but it should reset the state more cleanly.